src/components/override-sdk/field/AutoComplete/AutoComplete.tsx

```diff
@@ -85,6 +86,20 @@ export default function AutoComplete(props: AutoCompleteProps) {
   const actionsApi = thePConn.getActionsApi();
   const propName = thePConn.getStateProps()['value'];
 
+  useEffect(() => {
+    const script = document.createElement('script');
```

Comment:
can we create common util to add javascript file dynamically?

Reply:
will do as part of enhancement story in next sprint for autocomplete as im not able to test the changes for new cases in main branch as of now.